go.mod

```diff
@@ -12,6 +12,9 @@ require (
 	github.com/VividCortex/mysqlerr v1.0.0
 	github.com/apache/pulsar-client-go v0.11.0
 	github.com/aws/aws-sdk-go v1.44.259
+	github.com/aws/aws-sdk-go-v2 v1.19.1
```

Comment:
Why do we use `aws-sdk-go` v1 and v2 at the same time ? shall we only keep v2 ?

Reply:
BR package is using v1.